hri_safety_sense/src/JoystickHandler.cpp

```diff
@@ -71,20 +72,26 @@ uint32_t JoystickHandler::handleNewMsg(const VscMsgType &incomingMsg)
 		sensor_msgs::Joy sendLeftMsg;
 
 		sendLeftMsg.header.stamp = ros::Time::now();
-		sendLeftMsg.header.frame_id = "/srcs";
+		sendLeftMsg.header.frame_id = this->frameId;
 
-		sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftX));
-		sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftY));
-		sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftZ));
+		sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftX) /
+      this->AXIS_MAX);
```

Comment:
This indentation is a bit strange (same below several times).

More importantly, please use C++-style casts here (and below).

Reply:
5d0a284